x/programs/cmd/simulator/cmd/root.go

```diff
@@ -0,0 +1,80 @@
+// Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package cmd
+
+import (
+	"context"
+	"os"
+	"path"
+
+	"github.com/spf13/cobra"
+
+	"github.com/ava-labs/avalanchego/utils/logging"
+
+	"github.com/ava-labs/hypersdk/state"
+	"github.com/ava-labs/hypersdk/utils"
+)
+
+var (
+	dbFolder = ".simulator/db"
+	db       *state.SimpleMutable
+	dbCloser dBCloserFn
+
+	log logging.Logger
+)
+
+func NewRootCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "simulator",
+		Short: "HyperSDK program VM simulator",
+		PersistentPreRunE: func(*cobra.Command, []string) (err error) {
+			homeDir, err := os.UserHomeDir()
+			if err != nil {
+				return err
+			}
+
+			// Open storage
+			dbPath := path.Join(homeDir, dbFolder)
+			db, dbCloser, err = getStorage(context.Background(), dbPath)
+			if err != nil {
+				return err
+			}
+
+			log = logging.NewLogger(
+				"",
+				logging.NewWrappedCore(
+					logging.Info,
+					os.Stderr,
+					logging.Plain.ConsoleEncoder(),
+				))
+
+			utils.Outf("{{yellow}}database:{{/}} %s\n", dbPath)
```

Comment:
could we inject a timestamp into the logs? Or is that bad UX. To me the logs look funny without timestamps.

Reply:
all logs will use zap which has timestamps